packages/story-editor/src/app/media/local/useContextValueProvider.js

```diff
@@ -182,10 +189,33 @@ export default function useContextValueProvider(reducerState, reducerActions) {
     [setAudioProcessing, updateVideoIsMuted, removeAudioProcessing]
   );
 
+  const processMediaBaseColor = useCallback(
+    (resource) => {
+      const { baseColorProcessed, baseColorProcessing } = stateRef.current;
+      const { id } = resource;
+
+      const process = async () => {
+        // Simple way to prevent double-uploading.
+        if (
+          baseColorProcessed.includes(id) ||
```

Comment:
Wanted to test out how it works and did the following:
- Checked out the PR & refreshed the editor
- Dropped media from the media library to the Page as background (tried both with video and image)
- Saved and checked the preview -- didn't see the background color assigned (seemingly no base color was added in the front-end)
- Went back to the editor and refreshed the page
- Got the following error for this line:
<img width="680" alt="Screenshot 2021-11-09 at 13 34 15" src="https://user-images.githubusercontent.com/3294597/140916989-c6c6f630-9982-4180-bb2f-c15a0d6885d6.png">

Is there something I'm missing?

Reply:
Rebase issue. Can you test again?

Reply:
The base color is assigned now as expected, thanks!

Still seeing an error (a different one) when reloading the editor after dragging the media to background and saving:
<img width="1210" alt="Screenshot 2021-11-09 at 18 20 59" src="https://user-images.githubusercontent.com/3294597/140963173-e8a226b7-90d7-4ae4-b54b-4e41671d90b6.png">

Thoughts?

Reply:
@miina I am not seeing that. Try again now and ensure to get the latest version. 

If you are still seeing it, steps to replicate / video would be useful.

Reply:
@spacedmonkey Yep, still seeing. Here's a video:

https://user-images.githubusercontent.com/3294597/141096530-a7090118-5acb-4501-bf11-263b0e373989.mp4

Reply:
Checked again after new changes -- seems to be fixed now.